packages/core/src/components/SplitButton/__stories__/splitButton.stories.js

```diff
@@ -187,7 +188,7 @@ export const SplitButtonAsThePrimaryAction = {
   render: () => (
     <SplitButton
       children="Use template"
-      secondaryPositions={SplitButton.secondaryPositions.BOTTOM_END}
+      secondaryDialogPosition={DialogPosition.BOTTOM_START}
```

Comment:
You can use the secondaryDialogPosition from the SplitButton's static values. If I'm not wrong, we do not export the DialogPosition, so we want to show stories that users can just copy and use

Reply:
The issue is that the secondaryDialogPositions from the static values is of type SplitButtonSecondaryContentPosition and the secondaryDialogPosition is of type DialogPosition

Reply:
We resolved the issue by adding SplitButtonSecondaryContentPosition type to secondaryDialogPosition, and will remove the DialogPosition type in the next major release